src/actions/profile.action.js

```diff
@@ -39,6 +39,7 @@ const getProfile = () => {
         return toast.error('Please check your internet connection');
       }
       const user = decodeToken();
+      console.log(user, '....user');
```

Comment:
please remove the `console.log`

Reply:
> please remove the `console.log`

Thanks for spotting that out, I will work on it ASAP